eml/.eslintrc.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
     'indent': ['error', 2],
     'quotes': ['error', 'single'], // Specify single quotes
     'semi': ['error', 'always'], // Require semicolons
+    'no-unsafe-finally': 'off', // Disable no-unsafe-finally rule
```

Comment:
Why do we want to ignore unsafe controls in finally statements? I think it may be bad practice to ignore these

Reply:
This is because linting does not like the return statement and function call in the final block. The reason for this is that it will only throw the last error in cases where it throws multiple errors. This is fine, even intended.